### ISSUE TITLE
session-manager-plugin 1.2.295.0 SHA256 mismatch

### ISSUE THREAD

**Issue by tekumara** [Homebrew/homebrew-cask#121114]
### Verification

- [X] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [X] I have retried my command with `--force`.
- [X] I ran `brew update-reset && brew update` and retried my command.
- [X] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [X] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [X] I made doubly sure this is not a [checksum does not match](https://docs.brew.sh/Common-Issues#cask-checksum-does-not-match) error.

### Description of issue

```
$ brew install --cask session-manager-plugin
==> Downloading https://s3.amazonaws.com/session-manager-downloads/plugin/1.2.295.0/mac/session-manager-plugin.pkg
Already downloaded: /Users/tekumara/Library/Caches/Homebrew/downloads/456d9d21168cf84a91f8ac096612ada4ed42c61c569a92a2516ce0281c06dfa2--session-manager-plugin.pkg
Error: SHA256 mismatch
Expected: 67e527dd19b2ae6d5b5de62c75aedb4241f31b2ae177be3e118db8b4d067ad26
  Actual: dad3eb15607c675d20ddc8a4b199c46d458b70ed6188afb7c83fb1a1d7a44911
    File: /Users/oliver.mannion/Library/Caches/Homebrew/downloads/456d9d21168cf84a91f8ac096612ada4ed42c61c569a92a2516ce0281c06dfa2--session-manager-plugin.pkg
To retry an incomplete download, remove the file above.
```

The sha of the published binary has changed:
```
curl -s https://s3.amazonaws.com/session-manager-downloads/plugin/1.2.295.0/mac/session-manager-plugin.pkg | sha256sum
dad3eb15607c675d20ddc8a4b199c46d458b70ed6188afb7c83fb1a1d7a44911
```

But the latest version (as per [the docs](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-macos-signed)) has the expected sha:
```
curl -s https://s3.amazonaws.com/session-manager-downloads/plugin/latest/mac/session-manager-plugin.pkg | sha256sum
67e527dd19b2ae6d5b5de62c75aedb4241f31b2ae177be3e118db8b4d067ad26  -
```

Not sure the best course of action here?

### Command that failed

brew install --cask session-manager-plugin

### Output of command with `--verbose --debug`

```shell
N/A
```

### Output of `brew doctor --verbose`

```shell
N/A
```

### Output of `brew tap`

```shell
N/A
```

**Comment by carlocab** [comment]
https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match

**Comment by tekumara** [comment]
Thanks @carlocab I'm happy to raise a fix, but not sure exactly what to do here.

1. the cask version is 1.2.295.0 with sha `67e...`
2. the published binary is 1.2.295.0 with sha `dad3...`

So the published binary has been updated in place (although the binary with the cask sha is available from a different URL).

I can raise a fix to update to the sha, and keep the same cask version and URL, if that's most appropriate?

**Comment by bevanjkay** [comment]
@tekumara Yes that's the best way to move forward.
